lib/metalsmith.js

```diff
@@ -20,12 +21,45 @@ const views = [
   paths.govukfrontend
 ]
 
+const coloursAppliedSCSS = scssToJson('./node_modules/@govuk-frontend/globals/settings/_colours-applied.scss', {
+  dependencies: [{path: './node_modules/@govuk-frontend/globals/settings/_colours-palette.scss'}]
+})
+
+const coloursPaletteSCSS = scssToJson('./node_modules/@govuk-frontend/globals/settings/_colours-palette.scss')
+
+const coloursSCSS = Object.assign(coloursAppliedSCSS, coloursPaletteSCSS)
```

Comment:
Think this is worth a comment as it's not going to be obvious to everyone what this is doing.